docs/eventing/broker/broker-event-delivery.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Event delivery"
+weight: 04
+type: "docs"
+showlandingtoc: "false"
+aliases:
+  - /docs/eventing/event-delivery/#configuring-broker-delivery
+  - /docs/eventing/event-delivery/#broker-support
+---
+
+You can configure how events are delivered for each broker by adding a `delivery` spec to the `Broker` object, as shown in the following example:
```

Comment:
Figure out what we're doing about formatting API objects, maybe just leave this as broker in the meantime.